src/main/java/me/karakelley/http/Request.java

```diff
@@ -0,0 +1,62 @@
+package me.karakelley.http;
+
+import java.io.BufferedReader;
+import java.util.HashMap;
+
+public class Request {
+  private final RequestLine requestLine;
+  private final RequestHeaders requestHeaders;
+  private final BufferedReader reader;
+
+  public Request(BufferedReader reader) {
+    this.reader = reader;
+    this.requestLine = setRequestLine();
```

Comment:
It doesn't look like this method is actually setting anything but its name suggests otherwise.